src/common/entity/compute_state_domain.ts

```diff
@@ -1,6 +1,9 @@
 import { HassEntity } from "home-assistant-js-websocket";
 import computeDomain from "./compute_domain";
 
-export default function computeStateDomain(stateObj: HassEntity) {
+export default function computeStateDomain(stateObj: HassEntity | undefined) {
+  if (!stateObj) {
```

Comment:
We shouldn't put this check in here. It should not be called if there is no state obj.